closure/templates/closure_templates_plugin.bzl

```diff
@@ -0,0 +1,43 @@
+# Copyright 2019 The Closure Rules Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+SoyPluginInfo = provider(
+    doc = "provides the module and java_library implementing this plugin",
+)
+
+def _impl(ctx):
+    return [SoyPluginInfo(
+        generator = struct(
+            module = ctx.attr.module,
+            runtime = java_common.merge([dep[JavaInfo] for dep in ctx.attr.deps]),
+            deps = ctx.attr.deps,
```

Comment:
nit: This isn't needed anymore, we have everything in `runtime` now.

Reply:
Removed